app/config.py

```diff
@@ -85,6 +86,14 @@ def color_depth(
             return ColorDepth.DEFAULT
         return ColorDepth.TRUE_COLOR
 
+    @property
+    def width_100(self) -> Dimension:
+        return D(preferred=100 * 100)
+
+    @property
+    def height_100(self) -> Dimension:
+        return D(preferred=100 * 100)
+
```

Comment:
nothing to change here, i probably should have better commented my intention with these vars initially

basically these are just an easy way to force the child component to stretch to the width of its parent. The actual size value is meaningless - just a really large int, over the size of the terminal. 

nothing to change here, I think its totally fine, but just wanted to put it out there incase it changes the way we think about/organize these